modules/test/playwright/tests/frontend-data-set-admin-web/main/tests/data-set-admin/systemDataSets.spec.ts

```diff
@@ -260,28 +260,20 @@ test(
 			await creationModal.cancelButton.click();
 		});
 
-		await test.step('Item actions are imported with "detached" import policy', async () => {
+		await test.step('Advanced Sample item actions are imported with "item proxy" import policy', async () => {
```

Comment:
I agreed with BChan to use "items actions". He may catch this and the other one below :)